offer-maker/Forwarder.md

```diff
@@ -6,11 +6,11 @@ Forwarder should be seen as an implementation the can be used by multiple offer
 
 Forwarder does many of the same things as MangroveOffer and Direct with a few key differences.
 
-**How inbound tokens are deposited to the maker:** After the funds have been transferred from the taker to the Forwarder contract, it transfers the funds to the reserve of the offer owner. The reason for this, is that it cannot leave the funds on the contract, since the inbound tokens need to be distributed to each offer owners. When the offer has been successfully taken, it has no funds on the contract itself (vs. Direct where the funds are still on the contract). It therefore has no extra actions and just uses the default implementation of MangroveOffer.
+**How inbound tokens are handled (from the taker):** After the funds have been transferred from the taker to the Forwarder contract, it transfers the funds to the reserve of the offer owner. The reason for this, is that it cannot leave the funds on the contract, since the inbound tokens need to be distributed to each offer owners. When the offer has been successfully taken, it has no funds on the contract itself (vs. Direct where the funds are still on the contract). It therefore has no extra actions and just uses the default implementation of MangroveOffer.
 
-**How outbound tokens are deposited to the taker:** Since the contract cannot be used as the reserve for a Forwarder contract, this means that is also has to transfer the funds from the reserve to the Forwarder contract. This is again because multiple users can use the contract.
+**How outbound tokens are handled (for the taker):** Since the contract cannot be used as the reserve for a Forwarder contract, this means that is also has to transfer the funds from the reserve to the Forwarder contract. This is again because multiple users can use the contract.
 
-**Approvals:** When Forwarder checks whether the maker is allowed to set the a given address as its reserve, it first checks if the maker trying to set an empty address. This is allowed and the makers address will be used as the reserve. If the address is not empty, then it checks whether the address has approved the maker to use it as its reserve. This is different than Direct, since Forwarder has multiple makers, it has to keep track on which makers can use a reserve.
+**Reserve:** When Forwarder checks whether the maker is allowed to set the a given address as its reserve, it first checks if the maker trying to set an empty address. This is allowed and the makers address will be used as the reserve. If the address is not empty, then it checks whether the address has approved the maker to use it as its reserve. This is different than Direct, since Forwarder has multiple makers, it has to keep track on which makers can use a reserve. The reserve can call the Forwarder contract and approve or revoke approvals for an offer maker.
```

Comment:
Similarly to Direct, offer makers can change their reserve location in Forwarder. To do so they must prove they control the reserve by asking it to approve offer maker on Forwarder. Default reserve for an offer maker is the offer maker's address. Any reserve different than offer maker's address needs to approve Forwarder for outbound token transfer.

Reply:
Same as for Direct, added more description in MangroveOffer, and added  some of how Forwarder uses reserve.